### DISCUSSION TITLE
How's the progress of box32?

### DISCUSSION THREAD

**Discussion by xDoge26** [ptitSeb/box86#747] [Q&A]
Im just curious

**Comment by xDoge26** [comment]
Any news guys ?

**Reply by ptitSeb** [reply]
Nope, I'm still fixing and improving box86 & box64. I need solid version of both to really start box32. I haven't touched at my box32 for a few months, because box86/box64 is not stable enough and I cannot take them as reference.

**Reply by xDoge26** [reply]
I guess patience is key :)
